README.adoc

```diff
@@ -263,17 +255,28 @@ information that is received from the Server-Sent Events.
 The [hotspot=initSSE file=0]`initSSE()` method is called when the page first
 loads. This method subscribes the client to the SSE by creating a new instance of the
 [hotspot=eventSource file=0]`EventSource` interface and specifying the
-`http://localhost:9084/bff/sse` URL in the parameters. The [hotspot=eventSource
-file=0]`EventSource` interface makes a `GET` request to this endpoint with a
+`\http://localhost:9084/bff/sse` URL in the parameters. The [hotspot=eventSource file=0]`EventSource` 
```

Comment:
suggested edit:

To connect to the server, the [hotspot=eventSource file=0]`EventSource` 
interface makes a `GET` request to this endpoint with a
request header of `Accept: text/event-stream`.